app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -63,10 +63,10 @@ open class BpkSnapshotTest {
       .layout()
   }
 
-  protected fun snap(view: View) {
+  protected fun snap(view: View, tags: List<Any> = emptyList()) {
```

Comment:
this was implemented in the compose version, but never in the view version. re-added it here